Modules/_queuemodule.c

```diff
@@ -500,6 +500,21 @@ _queue_SimpleQueue_qsize_impl(simplequeueobject *self)
     return RingBuf_Len(&self->buf);
 }
 
+/*[clinic input]
+_queue.SimpleQueue.__sizeof__ -> Py_ssize_t
+
+Return size of queue in bytes, including underlying data structure.
```

Comment:
I'm not sure that this sentence is useful. It's implied by the sizeof protocol, no?